resources/qml/PrinterOutput/ManualPrinterControl.qml

```diff
@@ -101,29 +100,29 @@ Item
 
     Column
     {
-        enabled:
-        {
-            if (printerModel == null)
-            {
-                return false; //Can't control the printer if not connected
-            }
-
-            if (!connectedDevice.acceptsCommands)
-            {
-                return false; //Not allowed to do anything.
-            }
-
-            if(activePrintJob == null)
-            {
-                return true
-            }
-
-            if (activePrintJob.state == "printing" || activePrintJob.state == "resuming" || activePrintJob.state == "pausing" || activePrintJob.state == "error" || activePrintJob.state == "offline")
-            {
-                return false; //Printer is in a state where it can't react to manual control
-            }
-            return true;
-        }
+//        enabled:
```

Comment:
lots of commented-out code, which needs removing

Reply:
Actually it needed to be un-commented, it disabled the state checking for development to see the components while no USB printer was attached :P